src/components/NewTodo/NewTodo.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import { SelectOption } from '../SelectOption/SelectOption';
+import { TextField } from '../TextField/TextField';
+
+interface Props {
+  onChangeTodo: (title: string, userId: number, completed?: boolean) => void;
+}
+
+export const NewTodo: React.FC<Props> = ({ onChangeTodo }) => {
+  const [count, setCount] = useState(0);
```

Comment:
[CODE STYLE] - Don't use one-letter variable naming. Instead of 'count', consider using a more descriptive name that reflects its purpose in the context of the form, such as 'formKey' or 'resetCounter'.